stylesheets/_component.timeline.scss

```diff
@@ -0,0 +1,689 @@
+$timeline-content-border: #ccc;
+
+.has-timeline {
+    .content-main {
+        @include respond-max($screen-tablet) {
+            padding: 0;
+        }
+    }
+
+    .nav-inline {
+        margin: 0 $padding-base;
+
+        @include respond-min($screen-tablet) {
+            margin: 0;
+        }
+    }
+}
+
+.timeline {
+    margin: 0;
+    padding: 0;
+    position: relative;
+    word-break: break-word;
+
+    @include respond-min($screen-tablet) {
+        padding: 5px 0 2em;
+    }
+
+    @include ie-lte(8) {
+        &::before {
+            background: darken(#e9f0f5, 5%);
+            content: '';
+            height: 100%;
+            left: 35px;
+            position: absolute;
+            top: 54px;
+            width: 4px;
+        }
+    }
+
+    &::before {
+        @include respond-min($screen-tablet) {
+            background: darken(#e9f0f5, 5%);
+            content: '';
+            height: 100%;
+            left: 35px;
+            position: absolute;
+            top: 0;
+            width: 4px;
+
+            .lt-ie10 & {
+                top: 58px;
+            }
+        }
+    }
+
+    .autosave {
+        padding-top: .5em;
+    }
+}
+
+.timeline--no-messages::before {
+    @include respond-min($screen-tablet) {
+        content: none;
+    }
+}
+
+.timeline__notice {
```

Comment:
Again, this is specific to CXM timeline messages - are you other products using this?